test/parallel/test-debugger-invalid-json.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const common = require('../common');
+const startCLI = require('../common/debugger');
+
+common.skipIfInspectorDisabled();
+
+const assert = require('assert');
+const http = require('http');
+
+const host = '127.0.0.1';
+
+const testWhenBadRequest = () => {
+  const server = http.createServer((req, res) => {
+    res.statusCode = 400;
+    res.end('Bad Request');
+  });
+  server.listen(0, async () => {
+    try {
+      const port = server.address().port;
+      const cli = startCLI([`${host}:${port}`]);
+      const code = await cli.quit();
+      assert.strictEqual(code, 1);
+    } finally {
+      server.close();
+    }
+  });
```

Comment:
Using an async function could swallow errors, using `.then(common.mustCall(` allows to catch such errors.
```suggestion
  server.listen(0, common.mustCall(() => {
    const port = server.address().port;
    const cli = startCLI([`${host}:${port}`]);
    cli.quit().then(common.mustCall((code) => {
      assert.strictEqual(code, 1);
    }).finally(() => {
      server.close();
    });
  }));
```